script/publish_persistent_udfs

```diff
@@ -46,16 +46,16 @@ def main():
     args = parser.parse_args()
 
     raw_udfs = {x.name: x for x in read_udf_dir(args.udf_dir)}
+    published_udfs = []
```

Comment:
This is an efficiency tweak so that we deduplicate publishing of dependencies across _all_ udf definitions. Shortens run time a bit.